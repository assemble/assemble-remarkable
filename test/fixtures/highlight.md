# should highlight

```js
var foo = 'bar';
```
