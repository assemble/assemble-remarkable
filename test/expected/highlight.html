<h1>should highlight</h1>
<pre><code class="language-js"><span class="hljs-keyword">var</span> foo = <span class="hljs-string">'bar'</span>;
</code></pre>
